conf/defaults.ini

```diff
@@ -764,11 +764,24 @@ ha_gossip_interval = 200ms
 # The interval string is a possibly signed sequence of decimal numbers, followed by a unit suffix (ms, s, m, h, d), e.g. 30s or 1m.
 ha_push_pull_interval = 60s
 
+# Makes it possible to turn off alert rule execution but alerting UI is visible. If it's true (the default) then the respective legacy option is applied.
+execute_alerts = true
+
+# Default setting for alert calculation timeout. Default value is 30.  If it's 30 (the default) then the respective legacy option is applied.
+evaluation_timeout_seconds = 30
```

Comment:
Can we please change this value to use the same nomenclature as the other intervals? That means:

1. Dropping the `seconds` suffix
2. Then, adding the time parsing.